exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -123,7 +125,9 @@ drill.exec: {
     retry: {
       count: 7200,
       delay: 500
-    }
+    },
+    apply_secure_acl: false,
+    acl_provider: "creator-all"
```

Comment:
Please add these new configuration in **drill-override-example.conf** with a comment defining supported values of both configuration and what it means.

Reply:
Done